src/mongocxx/include/mongocxx/v_noabi/mongocxx/bulk_write.hpp

```diff
@@ -58,6 +58,13 @@ class bulk_write {
     ///
     ~bulk_write();
 
+    ///
+    /// Checks if a bulk write operation is empty.
+    ///
+    /// @return A boolean that returns true if the bulk write operation is empty.
+    ///
+    bool empty();
```

Comment:
```suggestion
    bool empty() const noexcept;
```

Use `noexcept` since no exceptions are thrown. Agrees with existing patterns in public methods.

Use `const` since object is not modified.